src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -65312,6 +65312,371 @@ static void Test()
             AssertDiagnosticOptions_36(source);
         }
 
+        [Fact]
+        public void DiagnosticOptions_62()
+        {
+            var source =
+@"
+#pragma warning disable nullable
```

Comment:
Please update the nullable speclet for #pragma change. 
It looks like previous #nullable change were also not captured.

Reply:
> Please update the nullable speclet for #pragma change. 
It looks like previous #nullable change were also not captured.

There is nothing to update in the speclet because it does not contain obsolete/invalid information. I see no good reason to reflect every implementation step in the speclet or duplicate specification in multiple places, implemented specification can be found here [https://github.com/dotnet/csharplang/blob/master/proposals/nullable-reference-types-specification.md](https://github.com/dotnet/csharplang/blob/master/proposals/nullable-reference-types-specification.md)  

---
In reply to: [245828755](https://github.com/dotnet/roslyn/pull/32208#discussion_r245828755) [](ancestors = 245828755)

Reply:
I forgot that Mads covered that in his spec. That works indeed

---
In reply to: [245830615](https://github.com/dotnet/roslyn/pull/32208#discussion_r245830615) [](ancestors = 245830615,245828755)